README.md

```diff
@@ -35,12 +35,23 @@ This module is optimally designed for use within a larger SIMP ecosystem, but it
 
 ### What useradd affects
 
-This module configures:
-  * `/etc/libuser.conf`
-  * `/etc/login.defs`
+This module can configure:
   * `/etc/default/nss`
   * `/etc/default/useradd`
+  * `/etc/group`
+  * `/etc/group-`
+  * `/etc/gshadow`
+  * `/etc/gshadow-`
+  * `/etc/libuser.conf`
+  * `/etc/login.defs`
+  * `/etc/passwd`
+  * `/etc/passwd-`
   * `/etc/profile.d/`
+  * `/etc/securetty`
+  * `/etc/security/opasswd`
+  * `/etc/shadow`
+  * `/etc/shadow-`
```

Comment:
The old source of these resources did. Maybe it was for the CVEs? https://github.com/simp/pupmod-simp-simplib/blob/5.X/manifests/init.pp#L286-L323